src/utils/components/Labels/LinuxLabel.tsx

```diff
@@ -0,0 +1,19 @@
+import React, { FC } from 'react';
+
+import { useKubevirtTranslation } from '@kubevirt-utils/hooks/useKubevirtTranslation';
+import { Label } from '@patternfly/react-core';
+import { LinuxIcon } from '@patternfly/react-icons';
+
+import './labels.scss';
+
+const LinuxLabel: FC = () => {
+  const { t } = useKubevirtTranslation();
+
+  return (
+    <Label variant="outline" className="title-label" icon={<LinuxIcon />}>
+      {t('Linux only')}
```

Comment:
I am not sure if it worth to create separate components for this. Maybe creating one compoment, maybe LabelOutline or something like that, and to get the text and the icon from props?

Reply:
As the label has unic text and icons I'm okay with that.
Having LabelOutline with the same text and icon (with its import) as props all around the repo is annoying for me

Reply:
Sometimes it is not so obvious where are the boundaries of what worth and what not, especially if we don't know how's gonna be in the future - that could help to make better decisions. Maybe there's gonna be many other similar use cases so we'd save some settings or css styling, maybe not. But for now yes, it probably doesn't worth.

But what to do anyway? This is IMO more like a question of an individual opinion. One or another approach isn't really bad. It will work without any issues or without any significant impact on the readability of the code or app performance. I'd simply go for the easier way, which is not creating a custom component.

But how this component was ever created? Let's understand this. In the earlier stages of this PR, there was a bit more stuff to put in the `OSLabel`. So it really seemed that it did worth to create it. But then I found how to simply add some more style for the Label component in this PR the way that we don't need extra elements to wrap the Label anymore, to achieve the expected look and especially extra space around the component. So those were removed, but the existence of the OSLabel remained. And now, we are simply in another stage, after the previous improvements, when we realize we probably don't even need OSLabel anymore, and can simply use the Label component as it is..